components/google_calendar/actions/add-attendees-to-event/add-attendees-to-event.mjs

```diff
@@ -1,10 +1,11 @@
 import googleCalendar from "../../google_calendar.app.mjs";
+import createEventCommon from "../common/create-event-common.mjs";
 
 export default {
-  key: "google_calendar-update-event-attendees",
-  name: "Update attendees of an event",
-  description: "Update attendees of an existing event. [See the documentation](https://googleapis.dev/nodejs/googleapis/latest/calendar/classes/Resource$Events.html#update)",
-  version: "0.1.4",
+  key: "google_calendar-add-attendees-to-event",
+  name: "Add Attendees To Event",
```

Comment:
I assume the change in the name means existing attendees are kept and not modified?

Reminder to redirect the old component key to the new one after this is merged